landoapi/landing_worker.py

```diff
@@ -69,6 +72,53 @@ def __init__(self, sleep_seconds=5):
         signal.signal(signal.SIGINT, self.exit_gracefully)
         signal.signal(signal.SIGTERM, self.exit_gracefully)
 
+    def _setup_ssh(self):
+        """Fetch a private SSH key from the environment and add it to ssh-agent.
+        """
+        SSH_PRIVATE_KEY_ENV_KEY = "SSH_PRIVATE_KEY"
+
+        # Fetch ssh private key from the environment. Note that this key should be
+        # stored in standard format including all new lines and new line at the end
+        # of the file.
+        ssh_private_key = os.environ.get(SSH_PRIVATE_KEY_ENV_KEY)
+        if not ssh_private_key:
+            logger.warning(f"No {SSH_PRIVATE_KEY_ENV_KEY} present in environment.")
+            return
```

Comment:
The worker does not care about the protocol. If provided an HTTPS path, even on production, it will still try to push (though obviously we don't have any repos like this right now).

Also, when running the worker locally or on the dev environments, not having an SSH key won't necessarily result in an error.

Thoughts?